Neos.Flow/Classes/Http/Client/InternalRequestEngine.php

```diff
@@ -126,8 +125,7 @@ public function sendRequest(RequestInterface $httpRequest): ResponseInterface
         }
 
         $requestHandler = $this->bootstrap->getActiveRequestHandler();
-        /** @phpstan-ignore-next-line composer doesnt autoload this class */
-        if (!$requestHandler instanceof FunctionalTestRequestHandler) {
+        if (!$this->bootstrap->getContext()->isTesting()) {
```

Comment:
In preparation of using the new request handler in the future, this is more generic. Although I think the whole condition is a bit overkill..

Reply:
Yeah, used in the behat tests I will see if/what we want to do there, but yes, if I get to this, it has to.